commcare_connect/opportunity/views.py

```diff
@@ -315,29 +320,91 @@ def form_valid(self, form):
         return response
 
 
-class OpportunityDetail(OrganizationUserMixin, DetailView):
+def is_program_manager_of_opportunity(request, opportunity):
+    return (
+        opportunity.managed
+        and opportunity.managedopportunity.program.organization == request.org.slug
+        and (request.org_membership.is_admin or request.user.is_superuser)
+    )
+
+
+class OpportunityDashboard(OrganizationUserMixin, DetailView):
     model = Opportunity
-    template_name = "opportunity/opportunity_detail.html"
+    template_name = "tailwind/pages/opportunity_dashboard/dashboard.html"
 
     def get(self, request, *args, **kwargs):
         self.object = self.get_object()
         if not self.object.is_setup_complete:
             messages.warning(request, "Please complete the opportunity setup to view it")
             return redirect("opportunity:add_payment_units", org_slug=request.org.slug, pk=self.object.id)
-        context = self.get_context_data(object=self.object)
+        context = self.get_context_data(object=self.object, request=request)
         return self.render_to_response(context)
 
-    def get_context_data(self, object, **kwargs):
+    def get_context_data(self, object, request, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["export_task_id"] = self.request.GET.get("export_task_id")
-        context["visit_export_form"] = VisitExportForm()
-        context["export_form"] = PaymentExportForm()
-        context["review_visit_export_form"] = ReviewVisitExportForm()
-        context["user_is_network_manager"] = object.managed and object.organization == self.request.org
-        import_visit_helper_text = _(
-            'The file must contain at least the "Visit ID"{extra} and "Status" column. The import is case-insensitive.'
-        ).format(extra=_(', "Justification"') if object.managed else "")
-        context["import_visit_helper_text"] = import_visit_helper_text
+
+        learn_module_count = LearnModule.objects.filter(app=object.learn_app).count()
+        deliver_unit_count = DeliverUnit.objects.filter(app=object.deliver_app).count()
+        payment_unit_count = object.paymentunit_set.count()
+
+        def safe_display(value):
+            if value is None:
+                return "---"
+            if isinstance(value, datetime.date):
+                return value.strftime("%Y-%m-%d")
+            return str(value)
+
+        path = [
```

Comment:
Directly set `context["path"]` etc to avoid redundant vars. Same for path, resources, basic_details